scene/2d/node_2d.cpp

```diff
@@ -169,11 +169,11 @@ void Node2D::set_scale(const Size2 &p_scale) {
 		((Node2D *)this)->_update_xform_values();
 	}
 	_scale = p_scale;
-	// Avoid having 0 scale values, can lead to errors in physics and rendering.
-	if (_scale.x == 0) {
+	// Avoid having 0 scale values, can lead to errors in physics and rendering. Limit scale to (+-)0.00001.
```

Comment:
The `CMP_EPSILON` constant is already used below. The change to the comment is not necessary and can be worse in the event that the constant is changed. I will give this a :heavy_check_mark: if it's just the code changes.

Reply:
I removed unnecessary addition to comment 👍